src/bin/customer/watch.rs

```diff
@@ -172,6 +179,10 @@ async fn dispatch_channel(
         close::claim_funds(database, config, &channel.label)
             .await
             .context("Chain watcher failed to claim funds")?;
+
+        // Developer note: if we separate the logic so that this is not always called immediately
+        // after the previous function, make sure it is still called in the case where the
```

Comment:
Could this explicitly say "close::claim_funds" rather than "the previous function"? :p